modules/charts/README.md

```diff
@@ -1,4 +1,20 @@
-- dev mode watches for file changes
-- outputs types, needs latest node/tsconfig setup in consumer project
-- package.json "files" set what's installed as dependency, package.json automatically included
-- `npm i <this_project_folder>` from consumer project
+# Introduction
+
+## Assumptions
+
+Arranger Charts uses <ArrangerChartsProvider> which internally uses Arranger Components hooks.
+Please make sure <ArrangerChartsProvider> is nested under an instance of <ArrangerDataProvider>
+
+This functionality is needed to query an Arranger server and to update SQON state in an Arranger application.
```

Comment:
Is there some default behaviour when one (or both) of the Providers are missing? Perhaps it puts an error message intended for the developer to communicate that a provider is missing? Should we document that default behaviour here?